src/pages/components/CertificateViewDetails.js

```diff
@@ -0,0 +1,215 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import CertificateTimeInterval from './CertificateTimeInterval'
+
+import BgIconDateSVG from '../../assets/images/icon-date.svg'
+import BgIconTimeSVG from '../../assets/images/icon-time.svg'
+
+export default function CertificateViewDetails({
+  size,
+  start,
+  end,
+  energy,
+  eco2,
+}) {
+  return (
+    <>
+      <PaddedWrapperDiv>
+        <ContainerFlexWrapDiv>
+          {start && end && (
```

Comment:
There's a number of usages of `&&` here to conditionally render content which is generally not great in react and especially bad when we have numbers (e.g. `eco2`). My tempatation is to refactor the conditionally rendered part to it's own component (which can be in this file) and then conditionally render with a ternary that ensures `null` correctly applied. Also in the case of numbers check the value is a number rather than for it's truthyness.

See https://react.dev/learn/conditional-rendering#logical-and-operator-

Reply:
Is this better?